src/view/com/auth/LoggedOut.tsx

```diff
@@ -19,15 +20,16 @@ enum ScreenState {
 export const LoggedOut = observer(function LoggedOutImpl() {
   const pal = usePalette('default')
   const store = useStores()
+  const {setMinimalShellMode} = useShellState()
```

Comment:
OK, so this pattern is a problem. We only need a setter but we're listening to the context which changes (and will trigger re-renders). So this and other components that only want to send messages like `setMinimalShellMode` are now subscribing to the actual shell mode changes. 

I propose that instead we separate two contexts and two values:

- `useShellState` contains UI stuff that changes (like `isDrawerOpen` and `minimalShellMode`)
- `useShellDispatch` exposes a stable `dispatch` method that accepts commands like `'openDrawer'`, `'closeDrawer'`, `setShellMode'`

I think you can implement this as a single `useReducer` instead of multiple state variables, and do

```
const [state, dispatch] = useReducer(shellReducer)

return (
  <ShellStateContext.Provider value={state}>
    <ShellDispatchContext.Provider value={dispatch}>
      ...
```